### DISCUSSION TITLE
How to contribute to ha-fusion

### DISCUSSION THREAD

**Discussion by janiskelemen** [matt8707/ha-fusion#117] [General]
**ha-fusion is currently in pre beta so things might change a lot at the moment**

If you want to contribute to ha-fusion and help to push it forward, it's crucial to follow some guidelines and best practices.

### 🐛 Bug reports
- Check if there is an open issue already before creating a new one
- Describe the issue as good as possible and mention the version you are on
- Add screenshots if possible
- Create a PR with a suggested fix if possible 

### 👨‍💻 How to Contribute
1. **Fork the repository**
   - Click on the 'Fork' button on the top right corner of the repository page on GitHub.
2. **Clone your fork**
   - On your GitHub account, navigate to the fork you just made and use the 'Clone' button to clone the repository to your local machine.
3. **Create a branch**
   - Switch to the directory on your machine where you've cloned the repo and use `git checkout -b your-new-branch-name`.
4. **Make your changes**
   - Make the necessary changes to the code base and don't forget to add or update tests as necessary.
5. **Commit your changes**
   - Commit your changes with a clear and detailed commit message, explaining the changes made and why.
6. **Push to your fork**
   - Push your changes to your fork on GitHub.
7. **Submit a Pull Request**
   - Navigate back to the original repository and click the 'Compare & pull request' button. Fill in a clear description of your changes. Screenshots and screencasts are very welcome!
8. **Wait for Review**
   - Maintain your pull request if there are any comments from reviewers.

### 🙏 Code of Conduct
- Be respectful and constructive.
- Harassment of any kind will not be tolerated.

**Comment by SoCuul** [comment]
@janiskelemen Wouldn't pre-beta be the same as alpha?
